tests/common/compare.py

```diff
@@ -42,6 +44,15 @@ def _check_item(item1: Any, item2: Any, path: str, rtol: float = 0.0, atol: floa
         assert isinstance(item2, type(item1)), f"{path} differs: {item1} != {item2}"
         _check_list_recursively(item1, item2, path, atol=atol, rtol=rtol)
         return
+    if isinstance(item1, Time):
+        assert type(item1) == type(item2)
+        assert item1 == item2, f"{path} differs: {item1} != {item2}"
+        return
+    if callable(item1):
+        assert callable(item2)
+        assert item1.__name__ == item2.__name__, f"{path} differs: {item1} != {item2}"
```

Comment:
This is an extremely loose equality check, open to suggestions.

Reply:
I don't have any great suggestions for the equality check, but do we want to
```
assert hasattr(item1, __name__)
assert hasattr(item2, __name__)
```
before comparing the items' `__name__`?

Reply:
Decided to remove this check, and not save the callable from `SWALR`